options.go

```diff
@@ -289,11 +286,12 @@ func coerce(v interface{}, opt interface{}, arg string) (interface{}, error) {
 	return nil, fmt.Errorf("invalid value type %T", v)
 }
 
-func hasArg(s string) bool {
-	for _, arg := range os.Args[1:] {
```

Comment:
In tests, I don't think this was ever actually working in most tests. I think they passed because they were using flagSet defaults instead.